crowdstrikeoauthapi.json

```diff
@@ -15,11 +15,11 @@
         }
     ],
     "license": "Copyright (c) 2019-2022 Splunk Inc.",
-    "app_version": "3.6.1",
+    "app_version": "3.6.0",
```

Comment:
The version should be bumped up instead of decrementing, right?

Reply:
the version got incremented accidentally. so I reverted it back to original 3.6.0

Reply:
I see. Thanks!